install_config/web_console_customization.adoc

```diff
@@ -855,6 +855,32 @@ xref:../install_config/configuring_authentication.adoc#OpenID[OpenID] identity
 providers, which require visiting an external URL to destroy single sign-on
 sessions.
 
+[[changing-the-membership-whitelist]]
+== Changing the Membership Whitelist
+
+The default whitelist in the membership page will show a certain subset of Cluster Roles
+(such as 'admin', 'basic-user', 'edit', etc), as well as any custom roles defined within
+a project.
```

Comment:
Why would someone want to change the membership whitelist? What's the use case for this? We should make that clear in the section intro.

Reply:
From the bugzilla above, this person seems to want to show ONLY custom roles, so no cluster roles.  I'd guess they would `MEMBERSHIP_WHITELIST: [ ]` to eliminate ALL cluster roles.  Perhaps they basically recreated the main set of `basic-user`, `view`, etc, but have some subtle nuance for each.  

I'm guessing here and betting there would be some pain points with this approach.